sdk/eventhub/Azure.Messaging.EventHubs/README.md

```diff
@@ -111,7 +111,7 @@ await using (var consumer = new EventHubConsumerClient(consumerGroup, connection
     using var cancellationSource = new CancellationTokenSource();
     cancellationSource.CancelAfter(TimeSpan.FromSeconds(45));
 
-    await foreach (PartitionEvent receivedEvent in consumer.ReadEvents(cancellationSource.Token))
+    await foreach (PartitionEvent receivedEvent in consumer.ReadEventsAsync(cancellationSource.Token))
```

Comment:
nit: implementing these as snippets would automatically fix these for you. 😄

Reply:
Yes, and it needs to be done.   It just hasn't bubbled to the top of the list yet.  :smile: